apps/fp-frontend-app/src/fagsak/components/SidePanelKnapp.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { SidebarRightIcon } from '@navikt/aksel-icons';
+import { Button } from '@navikt/ds-react';
+
+import styles from './fagsakGrid.module.css';
+
+interface Props {
+  toggleSideMeny: () => void;
+  visSideMeny: boolean;
+  style?: React.CSSProperties;
```

Comment:
Sidan du har delt knappen i to så kan du fjerna props visSideMeny (denne er alltid lik false), og style (Unødvendig å sende denne inn => bedre om style ligg direkte i SidePanelKnapp)